code/modules/mob/living/carbon/alien/humanoid/caste/ravager.dm

```diff
@@ -36,6 +37,8 @@
 					if(!step(L, dir))
 						break
 					sleep(1)
+			else
+				L.visible_message("<span class ='danger'>[src] is blocked by [L]'s shield!</span>")
```

Comment:
shields aren't the only thing able to block,there are armors too like riot armors i believe but shrug